composer.json

```diff
@@ -67,6 +67,12 @@
 		"check-cs": [
 			"@php ./vendor/squizlabs/php_codesniffer/scripts/phpcs"
```

Comment:
It seems that all references to `/scripts/` need to be changed to `/bin/` in the scripts.

Reply:
They can't be.

PHPCS 2.x used the `scripts` directory.
PHPCS 3.x uses the `bin` directory.

As Free is still on PHPCS 2.x and "Free" is the starting point for the `composer.json` file, by default, Composer scripts should use the PHPCS `scripts` directory.

Only the Premium script (targeted at PHPCS 3.x) should use the `bin` directory.
This should not give a problem with the `config-yoastcs` command as the `premium-check-cs` script does not call that (on purpose) as YoastCS 1.2.1 comes with the DealerDirect Composer PHPCS plugin which will automatically be run on the `composer update` to YoastCS 1.2.1 setting the PHPCS `installed_paths` to the right value.

Reply:
Just thinking - if this was in reference to the `fix-cs` script, then it might be worth it to add a separate `premium-fix-cs` script.

I didn't do so originally as the Premium repo is clean of all _fixable_ errors/warnings at this moment, but I can imagine it might be helpful for devs to be able to run it on their WIP branches for Premium, so if you like, I could add that script.

Reply:
That would be useful, let's do that separately.

Reply:
I'll happily add a second commit to this PR adding this.
And while I'm at it, I may as well also call the `config-yoastcs` script after the Premium reset.